trie/sync.go

```diff
@@ -80,7 +83,7 @@ func NewTrieSyncer(arg ArgTrieSyncer) (*trieSyncer, error) {
 		nodesForTrie:              make(map[string]trieNodeInfo),
 		topic:                     arg.Topic,
 		shardId:                   arg.ShardId,
-		waitTimeBetweenRequests:   time.Second,
+		timeBetweenRechecks:       arg.TimeBetweenRechecks,
```

Comment:
is this intended? moving from 1s to 10ms is a pretty big difference

Reply:
yes, v1 of the trie sync is not used for more than 1 1/2 years. Should have been deleted anyway

Reply:
To answer your question, this adjustment should make the v1 trie sync variant better